judge/models/tests/test_contest.py

```diff
@@ -0,0 +1,454 @@
+from django.core.exceptions import ValidationError
+from django.test import SimpleTestCase, TestCase
+from django.utils import timezone
+
+from judge.models import Contest, ContestParticipation, ContestTag
+from judge.models.contest import MinValueOrNoneValidator
+from judge.models.tests.util import CommonDataMixin, create_contest, create_contest_participation, create_user
+
+
+class ContestTestCase(CommonDataMixin, TestCase):
```

Comment:
From codecov analysis, you should test the `raise AttributeError` of `get_label_for_problem`.

Reply:
Even relooking that the `lupa` documentation, I can't find a way to cause `AttributeError` to be raised. This is because the argument passed in is an integer, so there are no attributes you can even access.

Reply:
Hm, I guess we have it there just in case. Sounds good though.